modules/serverless-openshift-routes.adoc

```diff
@@ -0,0 +1,98 @@
+// * serverless/networking/serverless-configuring-routes.adoc
+
+[id="serverless-openshift-routes_{context}"]
+= Configuring {product-title} routes for Knative services
+
+If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName}, and instead manually create a `Route` resource for the service.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving component must be installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that includes the `serving.knative.openshift.io/disableRoute=true` annotation:
++
+.Example YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+spec:
+  template:
+    metadata:
+      annotations:
+        openshift.io/disableRoute: true
+        # Check this with Kenjiro
+    spec:
+      containers:
+      - image: <image>
+----
++
+.Example `kn` command
+[source,terminal]
+----
+$ kn service create hello-example \
+  --image=gcr.io/knative-samples/helloworld-go \
+  --annotation serving.knative.openshift.io/disableRoute=true
+----
+
+. Verify that no {product-title} route has been created for the service:
++
+.Example command
+[source,terminal]
+----
+$ oc get routes.route.openshift.io -l serving.knative.openshift.io/ingressName=$KSERVICE_NAME -l serving.knative.openshift.io/ingressNamespace=$KSERVICE_NAMESPACE -n knative-serving-ingress
+----
++
+You should see the following output:
++
+[source,terminal]
+----
+No resources found in knative-serving-ingress namespace.
+----
+
+. Create a `Route` object in the `knative-serving-ingress` namespace by copying the following sample YAML and modifying the replaceable values:
++
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/timeout: 600s <1>
+  name: <route_name> <2>
+  namespace: knative-serving-ingress <3>
+spec:
+  host: <service_host> <4>
+  port:
+    targetPort: http2
+  To:
+    kind: Service
+    name: kourier
+    weight: 100
+  Tls:
+    insecureEdgeTerminationPolicy: Allow
+    termination: edge
+    key: |- <5>
+      -----BEGIN PRIVATE KEY-----
+      [...]
+      -----END PRIVATE KEY-----
+    certificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+    caCertificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE----
+  wildcardPolicy: None
+----
+<1> The timeout value for the {product-title} route. You should set the same value as the `max-revision-timeout-seconds` setting (`600s` by default).
+<2> The name of {product-title} route.
+<3> The namespace for the {product-title} route. This must be `knative-serving-ingress`.
+<4> The hostname for external access. You can set this to `<service_name>-<service_namespace>.<domain>` or your custom domain if you have configured one.
```

Comment:
```suggestion
<4> The hostname for external access. You can set this to `<service_name>-<service_namespace>.<domain>`.
```

The custom domain needs a combination of [DomainMapping](https://docs.openshift.com/container-platform/4.6/serverless/networking/serverless-domain-mapping.html).
I will send the PR once this PR was merged.